pkg/model/stack.go

```diff
@@ -16,7 +16,6 @@ type Stack struct {
 type StackParameters map[string]StackParameter
 
 type StackParameter struct {
-	Name         string  `json:"name"`
```

Comment:
You will need to adapt the transformation of paramters to JSON in the handler. Otherwise the frontend will not have the `name` in the JSON.